{{ cookiecutter.repo_name }}/README.md

```diff
@@ -145,6 +147,31 @@ no need to update the secrets baseline in this case.
 If your commit contains a mixture of false-positives and actual secrets, remove the actual secrets first before
 updating and auditing the secrets baseline.
 
+{% if cookiecutter.add_r_precommit_hooks == "Yes" %}
+***
+
+## Using this with R
+Please follow all the above steps.
+
+After that, we use [renv](https://github.com/rstudio/renv) to manage package dependencies here. Thus, before writing any code, run the following R code in your R session:
+
+```r
+# installs packages for pre-commit hooks in DESCRIPTION file
+renv::install()
+# installs packages for project captured in renv.lock
+renv::restore()
+```
+
+This will install additional packages that are necessary to work alongside the pre-commit hooks for R code.
+
+> Note, if you subsequently uninstall R after creating a project from this template with R hooks installed, then you will get the pre-commit hooks failing with `Executable Rscript not found`.*
+>
+> This is because the pre-commit hooks in `.pre-commit-config.yaml` require R packages. To remove this error message, manually remove the R pre-commit hooks from `.pre-commit-config.yaml` and re-run in your terminal, `pre-commit install`, to re-install your pre-commit hooks without the R-dependencies.
+
+{% endif -%}
+
+***
```

Comment:
There's a minor issue with the way this is parsed in Markdown if you select R pre-commit hooks. In the interests of time, I'll make this change and push it up. Will also remove the horizontal lines at the beginning and end of this section (one is inside the Jinja if statement, the other is outside).

Reply:
Fixed in commit [`3e5b47f`](https://github.com/ukgovdatascience/govcookiecutter/pull/9/commits/3e5b47f674870558cc04a0bc5fc34e85def12e2b).